src/components/AdvancedSelect.vue

```diff
@@ -365,25 +366,13 @@ export default {
      * Create a list of the filtered options; i.e. those that match the search
      */
     filtered() {
-      return this.linearOptions.filter(this.optionMatch);
+      return this.get_linear_options(this.filter_options(this.options));
     },
     /**
      * Create a linear list of all the options (headers included)
      */
     linearOptions() {
-      return this.options.reduce((f, o) => {
-        if (o.options) {
-          // push the header
-          f.push({
-            header: o.label,
-          });
-          f.push(...o.options.map(opt => Object.assign({}, opt, { parentHeader: o.label, selected: this.valueIsSelected(o.value) })));
-        } else {
-          // it's an item without group, push it to the list
-          f.push(Object.assign({}, o, { selected: this.valueIsSelected(o.value) }));
-        }
-        return f;
-      }, []);
+      return this.get_linear_options(this.options);
```

Comment:
It is used in the optionsMap computed property. I'll replace it there with the content of the linearOptions.

I replaced the computed property with a method because you can't use computed props with parameters.

Reply:
Updated